### PR TITLE
add test for penetrate()

### PR DESCRIPTION
**Pull request by EV9H**

### CODE REVIEW

File: tests/test_penetrate.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
+import pytest
+from src.team13LoremIpsum import penetrate
+from unittest.mock import patch, mock_open
+from rich.console import Console
+import pyfiglet
+
+
+class TestConsole:
```
Comment (bxw201) [inline tests/test_penetrate.py:8]: This test suite is currently testing the console package's code and not our own code

File: tests/test_penetrate.py [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+        expected_val = True
+        assert isinstance(console, Console) == expected_val
+
+    def test_console_print_color(self, console, capsys):
+        console.print("123")
+        captured = capsys.readouterr()
+        assert captured.out == "123\n"
+
+
+class TestPyfiglet:
```
Comment (bxw201) [inline tests/test_penetrate.py:24]: Same with this one. We should be calling our own written functions from the tests.

File: tests/test_penetrate.py [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+    def test_pyfiglet(self):
+        banner = pyfiglet.figlet_format("DARKWEB", font="banner3-D")
+        assert isinstance(banner, pyfiglet.FigletString) == True
+        assert len(banner) > 0
+
+
+class TestPenetrate:
```
Comment (bxw201) [inline tests/test_penetrate.py:31]: I think this is closer to what we're looking for. We need three tests per function. At this point, just to make the requirements in time, we should focus on testing specifically the penetrate function, perhaps for different inputs.

**Review by bxw201 — CHANGES_REQUESTED** [review]

**Review by bxw201 — APPROVED** [review]
lgm

**Review by Zackdan0227 — APPROVED** [review]

**Pull request merged**